ord_schema/units.py

```diff
@@ -70,15 +72,34 @@
     },
 }
 
+_FORBIDDEN_UNITS = {
+    'm': 'ambiguous between meter and minute',
+}
+
+# Concentration units are defined separately since they are not needed for any
+# native fields in the reaction schema.
+CONCENTRATION_UNIT_SYNONYMS = {
+    reaction_pb2.Concentration: {
+        reaction_pb2.Concentration.MOLAR: ['M', 'molar'],
+        reaction_pb2.Concentration.MILLIMOLAR: ['mM', 'millimolar'],
+        reaction_pb2.Concentration.MICROMOLAR: ['uM', 'micromolar'],
+    },
+}
+
 
 class UnitResolver:
     """Resolver class for translating value+unit strings into messages."""
 
-    def __init__(self):
+    def __init__(self, unit_synonyms=None, forbidden_units=None):
```

Comment:
Maybe use `forbidden_units=_FORBIDDEN_UNITS` as the default so `None` can mean that there are no forbidden units?

Reply:
I had that originally, but pylint yelled at me for using a dangerous default value since dicts are mutable. Would you prefer I just disable that pylint warning?

Reply:
No, I agree with that warning; requiring the user to pass in an empty dict is fine (please indicate that in the docstring).